### ISSUE TITLE
Endorsers interact with Attesters even without DAA

### ISSUE THREAD

**Issue by henkbirkholz** [ietf-rats-wg/draft-ietf-rats-reference-interaction-models#4]
Based on https://mailarchive.ietf.org/arch/msg/rats/okJriJPpapmZgeOfjbVGVP57bQk/

> This document extends the duties of the Endorser role as defined by
> the RATS architecture with respect to the provision of these Attester
> Identity documents to Attesters.  The existing duties of the Endorser
> role and the duties of a DAA Issuer are quite similar as illustrated
> in the following subsections.

Without DAA, I think the Endorser also needs to provision the Attester Identity to Attesters. And as I understand, the DAA Issuer is a supply chain entity before the Attester being shipped, so it is the Endorser when DAA is used, right? If yes, then in the next sentence, the comparison between Endorser and DAA Issuer doesn’t make sense to me.

**Comment by henkbirkholz** [comment]
tl;dr in essence, yes an Attester needs that communication arc anyways, but the provisioning is out-of-scope without the anticipated rechartering

In principle, that is correct, I think. An Attester is provisioned by the same authority (e.g. a supply chain entity) that is also in charge of the corresponding endorsements provided by the Endorser role. The "Provisioner" role does not necessarily have to reside on the same entity as the Endorser role, but both typically share the same owner and they also have to convey massages when they do not reside on the same entity (mainly the Claims for Endorsements that are based on the data provisioned on the Attester).

The RATS architecture does not include a corresponding arc between Endorser(/Provisioner) in https://www.ietf.org/archive/id/draft-ietf-rats-architecture-10.html#dataflow. During chartering of the WG the idea was to limit the scope of arcs between roles by excluding Attestation Provisioning Flows. Now, with the RATS architecture stabilizing, it becomes more apparent that there are pieces missing from that Figure 1. That is fine, as these can be described in other document.

DAA is a kind of an outlier that - if one looks at it closely - needs an Arc coming from that Provisioning scope even on the operational level that is in scope of the Attestation "Evidence" (and its friends) Flows. Does that shed a different light on the phrasing?

**Comment by Liqun-Chen** [comment]
The Endorser and DAA Issuer may not be the same entity and the DAA Issuer can provide the DAA credential to the Attester at anytime (in the supply chain or sometime later).

**Comment by henkbirkholz** [comment]
addressed in the new diagram in draft-birkholz-rats-daa
